openreader/docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: openreader_web_1
+      APP_PORT: 3003
+
+  web:
+    image: ghcr.io/richardr1126/openreader-webui:0.2.6@sha256:1f60d64898f85d24fab3a0b34be3c7c7a51b2fd37540ae54baca81d35e7289bb
+    user: "1000:1000"
+    environment:
+      - API_BASE=http://host.docker.internal:8880/v1 #Set the TTS API_BASE URL and/or API_KEY as the default for all devices. These values can also be overridden in the UI.

```

The other setting does not work. I set the API url of the kokoro container, this does the trick for me.
```suggestion
      - API_BASE=http://kokoro_web_1:8880/v1 #Set the TTS API_BASE URL and/or API_KEY as the default for all devices. These values can also be overridden in the UI.

```

---

I adapted it!